odl/tomo/backends/astra_setup.py

```diff
@@ -470,9 +523,9 @@ def astra_projector(vol_interp, astra_vol_geom, astra_proj_geom, ndim, impl):
     ndim = int(ndim)
 
     proj_type = astra_proj_geom['type']
-    if proj_type not in ('parallel', 'fanflat', 'parallel3d', 'cone',
-                         'cone_vec'):
-        raise ValueError('invalid 2d geometry type {!r}.'.format(proj_type))
+    if proj_type not in ('parallel', 'fanflat', 'fanflat_vec',
+                         'parallel3d', 'parallel3d_vec', 'cone', 'cone_vec'):
```

Comment:
Right. We don't use these ASTRA geometries anymore, but here we don't need to change it.